code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -159,7 +159,7 @@
 		shape.mind.transfer_to(stored)
 	if(death)
 		stored.death()
-	else if(convert_damage || istype(source) && source.convert_damage)
+	else if(convert_damage && istype(source) && source.convert_damage)
```

Comment:
This change doesn't make much sense, the original was wrong anyway.
It seems like it should be:
```
if(convert_damage || (istype(source) && source.convert_damage))
```

Reply:
I'll implement this.